Sunrise/ViewModels/OrdersViewModel.swift

```diff
@@ -74,10 +74,20 @@ class OrdersViewModel: BaseViewModel {
         }
     }
 
-    func orderOverviewViewModelForOrderAtIndexPath(indexPath: NSIndexPath) -> OrderOverviewViewModel {
-        let orderOverviewViewModel = OrderOverviewViewModel()
-        orderOverviewViewModel.order.value = indexPath.section == 0 ? orders[indexPath.row] : reservations[indexPath.row]
-        return orderOverviewViewModel
+    func orderOverviewViewModelForOrderAtIndexPath(indexPath: NSIndexPath) -> OrderOverviewViewModel? {
+        if indexPath.section == 0 {
+            let orderOverviewViewModel = OrderOverviewViewModel()
+            orderOverviewViewModel.order.value = indexPath.section == 0 ? orders[indexPath.row] : reservations[indexPath.row]
```

Comment:
The check for `indexPath.section == 0` will always return true here. Should probably look like the function below.